static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -56,17 +53,19 @@ const insertAdAtPara = (
 		});
 };
 
-const enableNearbyFilteringFix = () =>
-	!isInVariantSynchronous(spacefinderOkrMegaTest, 'control');
-
-const enableRichLinksFix = !isInVariantSynchronous(
-	spacefinderOkrMegaTest,
-	'control',
-);
-
-const filterNearbyCandidates = enableNearbyFilteringFix()
-	? filterNearbyCandidatesFixed
-	: filterNearbyCandidatesBroken;
+// this facilitates a second filtering, now taking into account the candidates' position/size relative to the other candidates
+const filterNearbyCandidates =
+	(maximumAdHeight: number) =>
+	(candidate: SpacefinderItem, lastWinner?: SpacefinderItem): boolean => {
+		if (
+			!lastWinner ||
+			Math.abs(candidate.top - lastWinner.top) - maximumAdHeight >=
+				adSlotClassSelectorSizes.minBelow
+		) {
+			return true;
+		}
+		return false;
```

Comment:
Could this be simplified slightly?

```suggestion
return (
	!lastWinner ||
	Math.abs(candidate.top - lastWinner.top) - maximumAdHeight >=
		adSlotClassSelectorSizes.minBelow
);
```

Reply:
Oooh good shout. 

I ended up with something close to yours but with an explicit early return if `lastWinner` is `undefined`:

https://github.com/guardian/frontend/pull/24938/files#diff-0ca555f0352210b70560a499ac7186955946730493d02e632bb8037e2ef8e9f0R57

This feels a bit easier to follow, what do you think?